app/main.py

```diff
@@ -2,0 +1,25 @@
+def copy_file(command: str) -> None:
+    if command.startswith("cp"):
+        command = command.replace("cp ", "")
+    else:
+        print("Unknown command")
+        return
+    list_of_files_name = command.split(" ")
+    if (list_of_files_name[0] != list_of_files_name[1]
+            and len(list_of_files_name) == 2):
+        try:
+            with (
+                open(list_of_files_name[0], "r")
+                as first_file,
+                open(list_of_files_name[1], "w")
+                as second_file
```

Comment:
The `with` statement is incorrectly formatted. The `open` statements should be correctly placed within the `with` block, using commas to separate them.